src/atoms/Tab/Tab.native.js

```
@@ -0,0 +1,134 @@
+import Icon from '../Icon';
+import Space from '../Space';
+import icons from '../../icons';
+import automation from '../../_helpers/automation-attributes';
+
+const styles = {
+  label: {
+    borderBottomWidth({ isActive }) {
+      if (isActive) {
+        return '1px';
+      }
+      return '0px';
+    },
+    borderBottomColor({ isActive, theme, disabled }) {
+      if (disabled) {
+        if (isActive) {
+          return getColor(theme, 'primary.930');
+        }
+      }
+      if (isActive) {
+        return getColor(theme, 'primary.800');
+      }
+      return 'transparent';
+    },
+    margin() {
+      return [0, 0, 0, 0.5];
+    },
+    color({ isActive, disabled, isPressed }) {
+      if (isPressed) {
+        if (isActive) {
+          return 'primary.900';
+        }
+        return 'shade.970';
+      } else if (isActive) {
+        return disabled ? 'primary.700' : 'primary.800';
+      } else {
+        return disabled ? 'shade.940' : 'shade.960';
+      }
+    },
+  },
+  tab: {
+    backgroundColor({ theme, isActive, isPressed }) {
+      if (isPressed) {
+        if (isActive) {
+          return getColor(theme, 'primary.920');
+        }
+        return getColor(theme, 'tone.940');
+      }
+      return 'transparent';
+    },
+  },
+};
+
+const StyledTabButton = styled(TouchableOpacity)`
+  min-width: 48px;
+  background-color: ${styles.tab.backgroundColor};
+  border-top-left-radius: 2px;
+  border-top-right-radius: 2px;
+`;
+
+const Label = styled(View)`
+  padding: 8px 0px;
+  border-bottom-width: ${styles.label.borderBottomWidth};
+  border-bottom-color: ${styles.label.borderBottomColor};
+`;
+
+const Tab = ({ label, isActive, icon, onPress, disabled, testID }) => {
+  const [isPressed, setTabPressed] = useState(false);
+
+  const onPressIn = useCallback(() => {
+    setTabPressed(true);
+  }, []);
+
+  const onPressOut = useCallback(() => {
+    setTabPressed(false);
+  }, []);
+
+  const isValidIcon = icons[icon];
```

Can we remove this check? We should let it crash, right?